Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1857,6 +1857,11 @@ extension BrowserViewController: TabManagerDelegate {
     func tabManager(_ tabManager: TabManager, didSelectedTabChange selected: Tab?, previous: Tab?, isRestoring: Bool) {
         libraryDrawerViewController?.close(immediately: true)
 
+        // Ecosia: fix crash when this is called but view is not loaded yet
+        if !isViewLoaded {
+            loadViewIfNeeded()
```

Comment:
Oh, restoration state strikes back

Reply:
yeah this could be the cause.. do you know a more elegant solution? @vauxhall 

BrowserViewController seems to be the only one that has `restorationIdentifier` set. Apparently this is not enough as also parents need this. I wonder if should deactivate state restoration overall. What do you think?

Reply:
I'm ok with deactivating it, I personally don't see much value for it, but we could ask product people what they think. 

This solution doesn't seem bad to me but also not sure it would be enough. We could also try to research a bit to make it safer, it has been a while since I used restoration.

In the old app, about 1.5 years we had restoration, we could also look at the git history how it was handled.

Reply:
hmm ok - I'll investigate a bit.. for now I'd say let's go with this patch only and if the crash persists, we can reiterate

Reply:
Sounds good to me 👍